server/views/layout/default.njk

```diff
@@ -16,10 +16,23 @@
     {% component "skip-link" %}
 
     {% set navLinks = [
-      { href: "/visit", title: "Visit us",        isCurrent: pageConfig.inSection === 'visit' },
-      { href: "/whats-on", title: "What's on",    isCurrent: pageConfig.inSection === 'whats-on' },
-      { href: "/explore", title: "Explore",       isCurrent: pageConfig.inSection === 'explore' },
-      { href: "/what-we-do", title: "What we do", isCurrent: pageConfig.inSection === 'what-we-do' }
+      {
+        href: "https://wellcomecollection.org/visit",
+        title: "Visit us"
+      },
+      {
+        href: "https://wellcomecollection.org/whats-on",
+        title: "What's on"
+      },
+      {
+        href: "/explore",
+        title: "Explore",
+        isCurrent: pageConfig.inSection === 'explore'
```

Comment:
The templates unfortunately don't have a method definition or a defined list of arguments (i.e. We can't say what arguments are required) - so not sending this over get's evaluated as `falsey`.

If we could specify a template function, I would have:
```
class NavLink(title: String, href: URL, isCurrent: Boolean = false)
def navigationTemplate(navLinks: List[NavLink])
```

I am looking at getting at least the server code typed (one of the reasons I am [removing WebPack](https://github.com/wellcometrust/wellcomecollection.org/pull/299)) which should help with some of this.

I could have left the code `isCurrent: pageConfig.inSection === 'what-we-do'` etc in - but felt it was misleading as that content doesn't exist.